### PR TITLE
Update

### PR DESCRIPTION
**Pull request by wolfv**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by sultaniman** [comment]
I accept maintainership.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,45 +1,47 @@
...
-    - python
+  host:
+    - python >=3.8,<4.0
+    - poetry-core >=1.0.0
     - pip
   run:
-    - python
-    - setuptools
+    - python >=3.8.0,<4.0.0
+    - types-setuptools >=69.1.0
```
Comment (sultaniman) [inline recipe/meta.yaml:24]: @wolfv should we lower this to 69.0.0.0 https://anaconda.org/conda-forge/types-setuptools?

Comment (sultaniman) [reply recipe/meta.yaml:24]: Just saw it got updated 🚀

**Comment by wolfv** [comment]
I am rerunning the CI now :)

**Pull request merged**